src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/custom_base.py

```diff
@@ -0,0 +1,132 @@
+
+
+def list_policies(client, resource_group_name, vault_name, workload_type=None, container_type="AzureIaasVM"):
+    if container_type == "AzureIaasVM" and workload_type is None:
+        return custom.list_policies(client, resource_group_name, vault_name)
+    else:
+        return custom_wl.list_wl_policies(client, resource_group_name, vault_name, workload_type, container_type)
+
+
+def show_item(cmd, client, resource_group_name, vault_name, container_name, name, workload_type=None, container_type="AzureIaasVM"):
+    if container_type == "AzureIaasVM" and workload_type is None:
+        return custom.show_item(cmd, client, resource_group_name, vault_name, container_name, name)
+    else:
+        return custom_wl.show_wl_item(client, resource_group_name, vault_name, container_name, name, workload_type)
+
+
+def list_items(cmd, client, resource_group_name, vault_name, workload_type=None, container_name=None, container_type="AzureIaasVM",
+               item_type="VM"):
+    if container_type == "AzureIaasVM":
+        return custom.list_items(cmd, client, resource_group_name, vault_name, container_name)
+    else:
+        return custom_wl.list_wl_items(client, resource_group_name, vault_name, workload_type, container_name)
+
+
+def list_recovery_points(cmd, client, resource_group_name, vault_name, container_name, item_name, workload_type=None,
+                         container_type="AzureIaasVM", item_type="VM", start_date=None, end_date=None):
+    if container_type == "AzureIaasVM" and workload_type is None:
+        return custom.list_recovery_points(cmd, client, resource_group_name, vault_name, container_name, item_name,
+                                           container_type, item_type, start_date, end_date)
+    else:
+        return custom_wl.list_wl_recovery_points(cmd, client, resource_group_name, vault_name, container_name, item_name, workload_type,
+                                                 start_date, end_date)
+
+
+def backup_now(cmd, client, resource_group_name, vault_name, container_name, item_name, retain_until,
+               container_type="AzureIaasVM", item_type="VM"):
+    if container_type == "AzureIaasVM":
+        return custom.backup_now(cmd, client, resource_group_name, vault_name, container_name, item_name, retain_until,
+                                 container_type, item_type)
+    else:
+        return custom_wl.backup_now()
+
+
+def disable_protection(cmd, client, resource_group_name, vault_name, container_name, item_name,
+                       container_type="AzureIaasVM", item_type="VM", delete_backup_data=False, **kwargs):
+    if container_type == "AzureIaasVM":
+        return custom.disable_protection(cmd, client, resource_group_name, vault_name, container_name, item_name,
+                                         container_type, item_type, delete_backup_data, **kwargs)
+    else:
+        return custom_wl.disable_protection()
```

Comment:
is this ongoing or there's no parameter being passed?

Reply:
Ongoing then but done now. Will be in tomorrow's commit